commissioning.tex

```diff
@@ -1,28 +1,46 @@
+
 
 \subsection{Template generation during commissioning}
 
-By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky. 
-Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science. 
+By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky.
+Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science.
 During LSSTCam commissioning we intend to incrementally generate templates over the maximal sky area supported by the available observations.
 
-The LSST SRD places well-defined criteria on the quality of the difference image and the amount of noise that a template can contribute to a difference image.  
-These criteria result in a minimum of three images being needed to construct a template for use in year one.  
-The commissioning period provides an excellent opportunity to investigate how many visits in a given band are sufficient to construct a usable template. 
-Given the desire to maximize the science harvest prior to the \drone,  relaxing these criteria is an option to be explored. 
+The LSST SRD places well-defined criteria on the quality of the difference image and the amount of noise that a template can contribute to a difference image.
+These criteria result in a minimum of three images being needed to construct a template for use in year one.
+The commissioning period provides an excellent opportunity to investigate how many visits in a given band are sufficient to construct a usable template.
+Given the desire to maximize the science harvest prior to the \drone,  relaxing these criteria is an option to be explored.
 
 \subsection{Alert generation during commissioning}
 
-Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period. 
-Where the accumulated ComCam data is sufficient for alert generation, we expect to provide alerts at high latency (weeks--months). 
+Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period.
+Where the accumulated ComCam data is sufficient for alert generation, we expect to provide alerts at high latency (weeks--months).
```

Comment:
I'd use some phrase other than "high latency".   "alerts only after a delay expected to be weeks to months".  Just English; no content change